openrarity/token/metadata/metadata.py

```diff
@@ -37,11 +37,12 @@ def validate_metadata(values):
         case "number":
             values["value"] = float(value)
         case "date":
-            values["value"] = (
-                value
-                if isinstance(value, (int | float))
-                else datetime.fromisoformat(value).timestamp()
-            )
+            if isinstance(value, (int | float)):
+                values["value"] = value
+            elif  isinstance(value, str) and value.replace(".", "",1).isdigit():
```

Comment:
Make sure you are using the pre-commit formatters that are included. It looks like the formatting on the 1 is missing a space. So just double check that your IDE is applying formatting on save and on commit. Otherwise, I think this is good.